pkg/routeagent_driver/handlers/kubeproxy/routes_iface.go

```diff
@@ -76,6 +76,11 @@ func (kp *SyncHandler) updateRoutingRulesForCIDRBlock(inputCidrBlock string, ope
 		if !kp.routeCacheGWNode.Has(inputCidrBlock) {
 			kp.routeCacheGWNode.Insert(inputCidrBlock)
 
+			if err := kp.configureRoute(inputCidrBlock, operation, viaGW); err != nil {
+				logger.Errorf(err, "Failed to add route %q for HostNetwork support on the Gateway node: %v",
+					inputCidrBlock, err)
+			}
+		} else if kp.localCableDriver == wireguard.GetName() {
```

Comment:
Is this appropriate for all calls to `updateRoutingRulesForCIDRBlock` and `updateRoutingRulesForHostNetworkSupport`, or only for the special case added in `LocalEndpointCreated`?

Reply:
Fair point, I reverted the changes in route_iface file, 
so, the current change will reconfigure host routes if  we are on GW node and cable driver is wireguard